.goreleaser.docker.yaml

```diff
@@ -280,6 +303,29 @@ dockers:
       - '--label=org.opencontainers.image.title=jvs-ui'
       - '--label=org.opencontainers.image.version={{ .Version }}'
 
+  -
+    ids:
+    - jvsctl
+    use: 'buildx'
+    goos: 'linux'
+    goarch: 'amd64'
+    image_templates:
+      - '{{ .Env.REGISTRY }}/prober:{{ .Version }}-amd64'
```

Comment:
FYI, you might need to resolve conflict since I merged https://github.com/abcxyz/jvs/pull/239, also please use DOCKER_TAG and .VERSION as a default, ```{{ if index .Env "DOCKER_TAG"  }}{{ .Env.DOCKER_TAG }}{{ else }}{{ .Version }}{{ end }}```

Reply:
I resolved the conflict, but forgot this. Just made the change. Thanks for the note!